bundles/org.openhab.binding.astro/src/test/java/org/openhab/binding/astro/internal/calc/SunCalcTest.java

```diff
@@ -0,0 +1,254 @@
+package org.openhab.binding.astro.internal.calc;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.openhab.binding.astro.internal.model.Sun;
+import org.openhab.binding.astro.internal.model.SunPhaseName;
+
+public class SunCalcTest {
```

Comment:
Please add a class level docblock, explaining the goal of this and mentioning the author

Reply:
The name of the class is ```SunCalcTest``` so it says that this is a general test of SunCalc. However the tests inside cover the case from issue #5006